shared/src/clients/phonebook-client.class/index.ts

```diff
@@ -59,4 +60,21 @@ export default class PhonebookClient {
       throw new Error(err as any)
     }
   }
+
+  public async getUniqueLinksForUsers(
+    body: PhonebookChannelDto
+  ): Promise<UserChannel[]> {
+    try {
+      const res = await this.request(
+        {
+          method: 'post',
+          url: 'public-user/unique_links',
```

Comment:
this reveals an inconsistency in phonebook api - follows both `-` and `_`.  Created as task https://linear.app/ogp/issue/PBOOK-35/inconsistencies-in-api-following-both-and-

Reply:
pre-emptively changing to use unique-links